pyriemann_qiskit/classification.py

```diff
@@ -940,10 +939,7 @@ def predict(self, X):
             print("Start NCH Predict")
         dist = self._predict_distances(X)
 
-        predictions = [
-            self.classes_[min(range(len(values)), key=values.__getitem__)]
-            for values in dist
-        ]
+        predictions = self.classes_[dist.argmin(axis=1)]
```

Comment:
The code fails here with error: 
*** AttributeError: 'list' object has no attribute 'argmin'
This is because dist is of type list and not np.array.

Reply:
Solved in last commits.